suites/pacific/cephadm/test-container-cli-args.yaml

```diff
@@ -0,0 +1,373 @@
+#===============================================================================================
+#-------------------------------------
+#----- Ceph code changes in 5.1 ------
+#--------------------------------------
+# * NFS deployment with pool and namespace is not supported.
+#
+#--------------------------------------
+#
+# Tier-level: 1
+# Test-Suite: test-container-cli-args.yaml
+# Test-Case: Deploy all services using a spec file option as below command
+#               eg., ceph orch apply -i <spec_file.yaml>
+#               included the spec option extra_container_args to change cpu limit
+#
+# Cluster Configuration:
+#    cephci/conf/pacific/cephadm/test-container-cli-args.yaml
+#
+#    4-Node cluster(RHEL-8.3 and above)
+#    3 MONS, 2 MDS, 1 MGR, 3 OSD and 2 RGW service daemon(s)
+#     Node1 - Mon, Mgr, Installer, OSD, alertmanager, grafana, prometheus, node-exporter
+#     Node2 - Mon, Mgr, OSD, MDS, RGW, alertmanager, node-exporter
+#     Node3 - Mon, OSD, MDS, RGW, node-exporter
+#     Node4 - Client
+#
+# Test Steps :
+#   (1) Bootstrap cluster with options,
+#       - skip-monitoring-stack: true
+#       - orphan-initial-daemons: true
+#       - registry-json: <registry-URL>
+#       - fsid: <cluster-fsid>
+#       - mon-ip: <monitor IP address: Required>
+#       - config: <ceph config options to be set during bootstrap>
+#   (2) Copy SSH keys to nodes.
+#   (3) Add nodes to cluster with address and role labels attached to it using Host spec yaml file.
+#   (4) Deploy services using apply spec option, (" ceph orch apply -i <spec_file>)
+#       - 3 Mon on node1, node2, node3 using host placements.
+#       - MGR using placement using label(mgr).
+#       - addition of OSD's using "all-avialable-devices" option.
+#       - alertmanager on node1, node2 using label "alert-manager".
+#       - grafana and prometheus on node1 using host placement with limit.
+#       - crash and node-exporter on all nodes using placement="*".
+#   (5) Deploy MDS service using spec file,
+#       (a) Create FS volume 'cephfs'
+#       (b) Deploy MDS with label(mds) option using 'cephfs' volume.
+#   (6) Deploy NFS service using spec file,
+#       (a) Create OSD pool 'nfs-rgw-pool'
+#       (b) Deploy NFS on node4 using 'nfs-rgw-pool' pool.
+#===============================================================================================
+tests:
+  - test:
+      name: Install ceph pre-requisites
+      desc: installation of ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+      config:
+        is_production: True
+  - test:
```

Comment:
would it make sense to combine bootstrap and host addition as one test case.

Reply:
Done.